src/Microsoft.Extensions.Logging.Configuration/baseline.netcore.json

```diff
@@ -31,111 +31,6 @@
         }
       ],
       "GenericParameters": []
-    },
-    {
-      "Name": "Microsoft.Extensions.Logging.Configuration.ILoggerProviderConfiguration<T0>",
```

Comment:
Isn't this a breaking change? Maybe @Eilon has more context?

Reply:
If the baseline appears to have a breaking change, does it even matter anymore? That would imply a break from 1.x to 2.0, so it's too late?

Reply:
SO, `ILoggerProviderConfiguration` exists in the release/2.1 branch but does not exist in either the release/2.0 or release/1.1 branch. I'm pretty sure that as mentioned above, this baselines was accidentally generated against 2.1.0, and therefore most changes to it listed here don't mean much.

Reply:
Ah yes that presumably explains it.